src/sciwrappers.c

```diff
@@ -227,6 +227,15 @@ void sci_add_text(ScintillaObject *sci, const gchar *text)
 	}
 }
 
+void sci_delete_char(ScintillaObject *sci, const gint pos)
+{
+	SSM(sci, SCI_DELETECHAR, pos, 0);
```

Comment:
why did you add this? Why not just use DELETERANGE with length 1?

Reply:
Yeah, same as before. Didn't know it.